source/application.py

```diff
@@ -0,0 +1,19 @@
+from flask import Flask
+from werkzeug.contrib.fixers import ProxyFix
+
+from endpoints.api import api
+from model.DeviceDAO import DeviceDAO
+from plugins.simpleLock.simpleLock import simpleLock
+from plugins.dimmableLight.DimmableLight import DimmableLight
+
+DAO = DeviceDAO()
+DAO.addDevice(simpleLock())
+#DAO.addDevice(DimmableLight())
```

Comment:
A dimmable light will be added but this was only as an example. It should ansd will e released